### PR TITLE
fix reflection warnings

### PR DESCRIPTION
**Pull request by vise890-ovo**
run `lein check` and fix resulting reflection warnings

### CODE REVIEW

**Comment by vise890-ovo** [comment]
i'm not sure the ci errors have got to do with this PR....

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 sudo: false
 language: clojure
-lein: lein2
+lein: lein
```
Comment (pyr) [inline .travis.yml:3]: why is this bundled here?

**Review by pyr — COMMENTED** [review]
I'll fix the CI errors separately, I'd like to go in without the travis change.

File: .travis.yml (hunk 1 above)
Comment (vise890-ovo) [reply .travis.yml:3]: ci wasn't passing so I made sure it wasn't anything I introduced

**Comment by vise890-ovo** [comment]
thanks for merging!

**Pull request merged**